microsetta_public_api/repo/_alpha_repo.py

```diff
@@ -1,11 +1,87 @@
+from microsetta_public_api import config
+from microsetta_public_api.exceptions import ConfigurationError, UnknownMetric
+from qiime2 import Artifact
+from q2_types.sample_data import AlphaDiversity, SampleData
+import os
+import pandas as pd
+
+
 class AlphaRepo:
 
+    # resources needs to be a class variable in order to be able to be
+    #  mocked in the test cases
```

Comment:
extra leading space

Reply:
I like leading spaces for continuation of comments. Helps differentiate comments if there are multiple, multi-line comments. Can change it if you twist my arm.

Reply:
mehhh... :) do as you please, thought it was a typo first time i saw but i see that now